### ISSUE TITLE
Batch add/remove users to sitegroup

### ISSUE THREAD

**Issue by michaelmaillot** [pnp/pnpjs#564]
### Category
- [ ] Enhancement
- [ ] Bug
- [x] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: 1.2.9

Please specify what version(s) of SharePoint you are targeting: Online

### Expected / Desired Behavior / Question
I want to add / remove multiples users to a Site Group through batch command.

### Observed Behavior
When I try to add / remove multiples users to a Site Group through batch command, nothing happens. Here's the code for add case:

```
const batch = this._currentWeb.createBatch();

this._currentWeb.siteGroups.getById(siteGroupId).users.inBatch(batch).add(user.id)
      .then((value: SiteUser) => {
        console.log(value);
      })
      .catch((error) => {
        console.log(error);
      });
    }

    await batch.execute()
    .then(() => {
      isOk = true
    })
    .catch((error) => {
      console.log(error)
    });
```

If I remove the "inBatch" part, it works both for adding and removing. Am I doing something wrong ?

Thank you for your help.

### Steps to Reproduce
N/A

**Comment by patrick-rodgers** [comment]
Just had a look, this is a bug in how the batch is getting processed. Working on a fix. Thanks!

**Comment by patrick-rodgers** [comment]
Ok, got a fix for this in #581
